config/routes.rb

```diff
@@ -2,5 +2,8 @@
   resources :reviews
   root 'restaurants#index', as: 'restaurant_index'
   resources :restaurants
+
+  post 'restaurants/load_data', to: 'restaurants#load_data'
```

Comment:
Note that you could put this under the 'restaurants' resource.

e.g.: 

```
resources :restaurants do
  post 'load_data', action: 'load_data'
end
```

And possibly you don't even need the action parameter - try and check documentation :) .